tests/test_multiprocessing.py

```diff
@@ -14,19 +14,17 @@
 limitations under the License.
 """
 import os
-from pathlib import Path
 
 import pytest
 
 import generative_data_prep
 from generative_data_prep.data_prep import multiprocess_data_prep
-
-from .test_utils import EXAMPLE_PATH
+from tests.conftest import TESTS_EXAMPLES_PATH
 
 
 def get_split_dir(test_name: str) -> str:
     """Create a absolute path to example input."""
-    return os.path.join(Path.cwd(), EXAMPLE_PATH, test_name, f"pipelined_{test_name}", "splits")
+    return TESTS_EXAMPLES_PATH / test_name / f"pipelined_{test_name}" / "splits"
```

Comment:
Should we use / instead of os.path.join ? Would that break on some OS?

Reply:
`Path` is the most robust way to use define and access paths in Python; this should be compatible with all libraries